runtime-modules/content/src/lib.rs

```diff
@@ -1660,7 +1660,7 @@ decl_module! {
 
                     (
                         updated_nft,
-                        RawEvent::BidMadeCompletingAuction(participant_id, video_id, None),
+                        RawEvent::BidMadeCompletingAuction(participant_id, video_id),
```

Comment:
@zeeshanakram3 I don't think your work on QN should impact those events in any way? Some rebase error or something?

Reply:
You are right. Since, I based this PR on #3513, I will update this from Rhodes once #3513 has been merged.